builtin.py

```diff
@@ -23,28 +26,33 @@ def on_select(self):
     def __str__(self):
         return str(self.from_node) + " -> " + str(self.to_node)
 
+
 # Breaking connections can only be used once
 class BreakingConnection(Connection):
     def __init__(self, from_node: Node, to_node: Node):
         super().__init__(from_node, to_node)
-        
+
     def on_select(self):
         self.from_node.connections.remove(self)
 
     def __str__(self):
-        return str(self.from_node) + " \-> " + str(self.to_node)
+        return str(self.from_node) + " \\-> " + str(self.to_node)
 
 
 # Base node classes
 
 class Node:
-    def __init__(self, title: str, desc: str):
+    def __init__(self, title: str, desc: str, coins: str):
+        super().__init__()
```

Comment:
we don't need super().__init__(), because this class is not inheriting anything (It's a base class)